### PR TITLE
Stack and queue

### PR DESCRIPTION
**Pull request by Tw1stedRain**

### CODE REVIEW

File: src/test/java/stacksAndQueues/QueueTest.java [hunk 1]
```diff
@@ -0,0 +1,79 @@
+package stacksAndQueues;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class QueueTest {
+
```
Comment (JessLovell) [inline src/test/java/stacksAndQueues/QueueTest.java]: Add a test method where you enqueue and dequeue interchangeably and peek. See if it is all working together as expected. Add a test that where you enqueue then DQ til the list is empty then enqueue again.

File: src/test/java/stacksAndQueues/QueueTest.java [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+        assertEquals(q.front.value, 2);
+        assertEquals(q.back.value, 6);
+
+        assertEquals(q.dequque(), 2);
+        assertEquals(q.dequque(), 4);
+        assertEquals(q.dequque(), 8);
+    }
+
+    @Test
+    public void testPeek() {
```
Comment (JessLovell) [inline src/test/java/stacksAndQueues/QueueTest.java]: This test is well written! 👍

File: src/test/java/stacksAndQueues/StackTest.java [hunk 3]
```diff
@@ -0,0 +1,70 @@
+package stacksAndQueues;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class StackTest {
+
```
Comment (JessLovell) [inline src/test/java/stacksAndQueues/StackTest.java]: Add a test method where you push/pop interchangeably and peek. See if it is all working together as expected.
